src/openrct2/core/Imaging.cpp

```diff
@@ -221,7 +221,8 @@ namespace Imaging
             if (image.Depth == 8)
             {
                 png_byte transparentIndex = 0;
-                png_set_tRNS(png_ptr, info_ptr, &transparentIndex, 1, nullptr);
+                if ((*image.Palette)[static_cast<uint16_t>(0)].Alpha < 255)
```

Comment:
why this change?

Reply:
Custom palettes can choose an opaque or transparent background

Reply:
What relevance does that have to creating a png?

Reply:
If the user wants an opaque background color for their exported png, they can get an opaque background color by setting the first pixel of the palette to opaque